send-transaction-service/src/send_transaction_service.rs

```diff
@@ -181,6 +184,150 @@ where
     }
 }
 
+/// Metrics of the send-transaction-service.
+#[derive(Default)]
+struct SendTransactionServiceStats {
+    /// Count of the received transactions
+    received_transactions: AtomicU64,
+
+    /// Count of the received duplicate transactions
+    received_duplicate_transactions: AtomicU64,
+
+    /// Count of transactions sent in batch
+    sent_transactions: AtomicU64,
+
+    /// Count of transactions not being added to retry queue
+    /// due to queue size limit
+    retry_queue_overflow: AtomicU64,
+
+    /// retry queue size
+    retry_queue_size: AtomicU64,
+
+    /// The count of batches
+    send_count: AtomicU64,
+
+    /// Time spent on transactions in non-batch mode in micro seconds
+    send_us: AtomicU64,
```

Comment:
Ha, sometimes the `us` abbreviation is really unfortunate. Nothing to do here; I just think `send_us` is funny

Reply:
Yes!